{{ cookiecutter.repo_directory }}/index.html

```diff
@@ -8,7 +8,7 @@
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<meta name="description" content="Bootstrap 5 Tech Conference Template">
+	<meta name="description" content="{{ cookiecutter.banner.description }}">
 	<meta name="author" content="Xiaoying Riley at 3rd Wave Media">
```

Comment:
While you're at this. Should author be changed to the hackweek organization?

Reply:
I was contemplating whether to change that and I think you are right. We still give template credit in the footer.
- [ ] Update author